### ISSUE TITLE
TODO for paper #4 :  Re-train top-performing model with ENCODE weight inits 

### ISSUE THREAD

**Issue by annashcherbina** [kundajelab/DMSO#13]
Compare to performance on random initializations.

**Comment by annashcherbina** [comment]
[dmso_bassett_classification_model_random_init_vs_ENCODE_init.xlsx](https://github.com/kundajelab/DMSO/files/1439192/dmso_bassett_classification_model_random_init_vs_ENCODE_init.xlsx)
![image](https://user-images.githubusercontent.com/5261545/32347460-f233872c-bfcd-11e7-8da6-b914008f76dd.png)

**Comment by annashcherbina** [comment]
I have fixed the batch normalization bug. 
I am now seeing the improvement with ENCODE weight initializations -- though it averages out to only 1% improvement on auPRC. I think that doesn't tell the full story though.

### RecallAtFDR50
there are 4 tasks that have validation split values of 0 with the base init but get over 0.4 with the ENCODE init. The overall average improvement on the validation split is 0.26 (.33 for random init, 0.59 for ENCODE init). 

![image](https://user-images.githubusercontent.com/5261545/32575213-448befc8-c488-11e7-85b3-bcc78e3dbbd7.png)

### auROC  
mean improvement  = 0.06 (0.79 for random init, 0.85 for ENCODE init): 
![image](https://user-images.githubusercontent.com/5261545/32575319-809ea8a2-c488-11e7-8dff-11541728ba08.png)


### auPRC 
mean improvement = 0.01 (0.54 for random init, 0.55 for ENCODE init): 
![image](https://user-images.githubusercontent.com/5261545/32575372-aa35be62-c488-11e7-91b7-5fa714f501c0.png)


### All task values: 

![image](https://user-images.githubusercontent.com/5261545/32575401-b9b09d26-c488-11e7-855d-976e364bbe26.png)


I am puzzled why my validation performance is higher than my training performance across all metrics. The validation loss is considerably higher than the training loss (0.95 for training vs 12.4 for validation)

**Comment by annashcherbina** [comment]
I know we discussed that it's not valid to average curves, so we can disregard the black central lines on these graphs, but these are the ROC & PRC curves for ENCODE init vs Random init (each colored line represents the ROC / PRC for a given task): 
![dmso encode_init train](https://user-images.githubusercontent.com/5261545/32575686-866a6d74-c489-11e7-8ec2-f84e2cca17b0.png)
![dmso encode_init valid](https://user-images.githubusercontent.com/5261545/32575693-8abfe4da-c489-11e7-9520-d9fe82aecf92.png)
![dmso rand_init train](https://user-images.githubusercontent.com/5261545/32575695-8d886930-c489-11e7-924b-92bd0141b990.png)
![dmso rand_init valid](https://user-images.githubusercontent.com/5261545/32575700-9056c814-c489-11e7-82a9-c902cd011a20.png)

**Comment by akundaje** [comment]
Nice. Those are big jumps where it matters the most ie. Recall at specific
FDRs
